src/mixins/nft.js

```diff
@@ -326,6 +326,17 @@ export default {
         : '-';
     },
     nftPaymentPriceInUSD() {
+      if (this.nftIsNFTBook) {
+        const result = this.getNFTBookStorePricesByClassId(this.classId);
+        if (!result || !result.length) return undefined;
+
+        const [price, ...prices] = result;
+        return prices.reduce(
+          (acc, price) => Math.min(acc, price.price),
```

Comment:
Not sure why linter doesnt warn, but price in array and price in arrow fn shadow each other